web/package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "axios": "^0.17.1",
     "babel-polyfill": "^6.26.0",
+    "fsevents": "^1.1.3",
```

Comment:
`fsevents` probably shouldn't be in our list of dependencies.  It should get pulled in by our dependencies on an as-needed basis.  It only works on macOS, so the build is failing (and the deploy would fail too) because we declare that we require it but the build is in a Linux container.

Reply:
ahhhh, thx!